app/lib/webhook-types.ts

```diff
@@ -8,6 +8,8 @@ export type EventType =
 	| "task.deleted"
 	| "task.status_changed"
 	| "task.assigned"
+	| "task.pr_opened"
+	| "task.pr_merged"
```

Comment:
I'm concerned our list of events are growing too much. It causes too much noise in the discord already